cmake/templates/cvconfig.h.in

```diff
@@ -59,6 +59,9 @@
 /* Vulkan support */
 #cmakedefine HAVE_VULKAN
 
+/* Webnn support */
+#cmakedefine HAVE_WEBNN
+
```

Comment:
?

Reply:
Hi @alalek, since I removed the ```#cmakedefine HAVE_WEBNN``` from the cvconfig.h.in file, the ```HAVE_WEBNN``` macro does not work well. Besides, I didn't find a proper .in file that only for dnn module to place this macro. Could you please give some suggestions? Thanks!

Reply:
There are no module-specific .in files for now.
Please add definition through CMake: https://github.com/opencv/opencv/blob/4.5.4/modules/dnn/CMakeLists.txt#L21-L23

Reply:
Thanks a lot! This has been fixed now.